.taskcluster.yml

```diff
@@ -5,6 +5,7 @@
 version: 1
 reporting: checks-v1
 policy:
+    allowComments: collaborators
```

Comment:
Sadly, for each of the variables in the `$let` clause below, we're going to need to set a value for the `github-issue-comment` event.

I recently did this a second time in the `fxci-config` repo (unlanded):
https://github.com/mozilla-releng/fxci-config/pull/155/commits/1c80e96aa488f606d839d1308fe1fc2cf5e0362f?diff=unified&w=1

I recommend setting an `isPullRequest` variable in an outer `$let` statement (along with associated actions there). Note that this will indent the entire rest of the file by another level which is unfortunate.

You can convert these variables to `$switch` statements while you're at it which is nicer to read imo.

Reply:
You'll also need to adjust the `$if` guard which decides whether to create the Decision task at all or not (near line 83 in this patch).